### PR TITLE
Actualize library

### PR DESCRIPTION
**Pull request by erickskrauch**
Added missing methods.
Fixed files uploading methods.
Removed lodash dependency.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -128,13 +141,13 @@ class CrowdinApi {
     }));
   }
 
-  getStream(path) {
+  getStream(path, qs = {}) {
     return handleStream(request.get({
       uri: this.uri(path),
-      qs: {
+      qs: Object.assign(qs, {
```
Comment (holm) [inline index.js]: How about using destructuring here also?

```
qs: {
  ...qs,
  json: true,
  key: this.apiKey
}
```

File: index.js [hunk 2]
```diff
@@ -91,6 +90,20 @@ async function handleStream(request) {
...
 }
 
+function packFiles(files) {
+  return Object.keys(files).reduce((acc, crowdinPath) => {
+      let value = files[crowdinPath];
+      if (typeof value === 'string') {
+          value = fs.createReadStream(value);
+      }
+
+      return ({
```
Comment (holm) [inline index.js]: It doesn't really matter much, but it's inefficient to create a new object and destructure the old one every iteration.

Could you not do:

```
acc[`files[${crowdinPath}]`] = value;

return acc;
```

File: index.js (hunk 1 above)
Comment (erickskrauch) [reply index.js]: I wanted to keep the same style as all other methods. I'll fix it now.

File: index.js (hunk 2 above)
Comment (erickskrauch) [reply index.js]: Redux has gotten into my brain too much 😅

**Comment by holm** [comment]
Thanks for your work on this. I fixed a few linting issues, bumped dependencies and released new version as 3.0.0.

**Pull request merged**